src/index.tsx

```diff
@@ -1,3 +1,6 @@
+export * from './Dropdown';
```

Comment:
```suggestion
export type { TriggerProps, DropdownProps } from './Dropdown';
```

Do not use `export *`.

Reply:
@afc163 sounds better, updated PR with this suggestion